src/components/common/Tabs/Tabs.tsx

```diff
@@ -25,24 +25,22 @@ const Wrapper = styled.div`
   }
 `
 
-const Tabs: React.FC<Props> = (props) => {
-  const [activeTab, setActiveTab] = useState<number>(1)
+const Tabs: React.FC<Props> = ({ tabItems }) => {
+  const [activeTab, setActiveTab] = useState(1)
 
   return (
     <Wrapper>
       <ul>
-        {props.tabItems.map(({ id, title, activeColor }) => (
+        {tabItems.map(({ id, title, activeColor }) => (
           <TabItem
             key={id}
-            id={id}
             title={title}
             onTabClick={(): void => setActiveTab(id)}
-            isActive={activeTab === id}
-            activeColor={activeColor}
+            activeColor={activeTab === id && activeColor}
```

Comment:
I wonder:
- We conditionally set the activeColor here? If activeColor has no value/false then this won't pass?
Right now if no `activeColor` is passed for a tabitem, it won't have a color at all. So likely need to add logic to fallback to a 'default'.

Btw that was the reason I wanted to get an 'isActive' to conditionally apply styles. But I understand we can still do this, but without having to pass a prop for it.

Reply:
Ah it already falls back anyhow in the styles after double checking. All good.